game/database/domains/action/BOLT.json

```diff
@@ -1,10 +1,4 @@
 {
-  "effects":[{
-      "typename":"deal_damage",
-      "target":"val:target_body",
-      "amount":"val:dmg"
-    }],
-  "cost":14,
   "operators":[{
       "typename":"get_body_at",
```

Comment:
next time, please try to add to the commit only the files you actually changed in the DB. This might generate unnecessary conflicts with other branches that do have important changes in these files that here are just shuffled, but otherwise unaltered in nature.